.github/workflows/ci.yml

```diff
@@ -299,3 +299,42 @@ jobs:
         run: pip install . -c ci/constraints-test/py310-min-install.txt -r reqs/requirements-dev-test.txt
       - name: Run the tests
         run: invoke tests --unit
+
+  import_gx:
+    needs: [unit-tests, static-analysis]
+    if: github.event.pull_request.draft == false
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.8"
+          cache: "pip"
+          cache-dependency-path: requirements.txt
+      - name: Install Great Expectations with core deps only
+        run: pip install .
+      - name: Import Great Expectations
+        run: python -c "import great_expectations as gx; print('Successfully imported GX Version:', gx.__version__)"
+
+  usage_stats_integration:
+    needs: [unit-tests, static-analysis]
+    if: github.event.pull_request.draft == false
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.8"
+          cache: "pip"
+          cache-dependency-path: requirements.txt
+      - name: Install Great Expectations with core deps only
+        run: pip install --constraint constraints-dev.txt ".[test]" pytest-azurepipelines
+      - name: Import Great Expectations
```

Comment:
This name should be updated to "Usage Stats Integration Test" or something similar.

Reply:
Woops, fixed.